desktop_version/src/Font.cpp

```diff
@@ -1049,7 +1049,14 @@ std::string string_unwordwrap(const std::string& s)
             else if (consecutive_newlines == 1)
             {
                 // The last character was already a newline, so change it back from the space we thought it should have become.
```

Comment:
Just making sure the comment still makes sense... (In two parts because suggestions work that way apparently)

```suggestion
```